docs/index.rst

```diff
@@ -3,17 +3,19 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to MeteoSwiss DevOps's documentation!
+Welcome to C2SM Spack Documentation!
 =============================================
 
```

Comment:
I would add a general description sentence like:
Spack is the package manager used by C2SM and MeteoSwiss to install and deploy the software our software on supercomputer mostly at the Swiss Super Computing Center (CSCS).